data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,2403 @@
+	on accept
+		payment 2000000
+		outfit "Enforcer Riot Staff" -10
+		dialog `The Unfettered thank you for bringing these 10 staffs to them. They give you <payment>.`
+
+mission "Bring Void Rifle"
+	invisible
+	source "Warfeed"
+	to offer
+		has "Unfettered: Close Quarter: offered"
+	on offer
+		require "Void Rifle" 1
+		conversation
+			`He Yi seems very, very interested in this weapon. "Where did you find it? Can you bring us more? We would gladly make you rich if you do, this is the perfect weapon for training."`
+			choice
+				`	"Sorry, this is one of a kind."`
+			`	"Oh, if you want I can cancel the exchange then, if it is precious to your eyes."`
+			choice
+				`	"<payment> is much more precious, thank you."`
+				`	"Yes I have changed my mind, cancel the exchange."`
+					decline
+			`	"This will be my personal weapon, I will take good care of it worry not!"`
+				accept
+	on accept
+		payment 10000000
+		outfit "Void Rifle" -1
+
+
+
+mission "Unfettered: Start Of The War"
+	name `Strength Of A Leader`
+	description `Prove your strength against Si-Ra and Hi-Ye in a battle. But do not kill them.`
+	source "Darkcloak"
+	to offer
+		has "event: shield beetle mass production"
+		has "event: new cooling system"
+		has "Unfettered: The Word Has Spread: done"
+		has "main plot completed"
+	on offer
+		conversation
+			`You find Si-Ra and Hi-Ye waiting for you on the spaceport. Si-Ra starts speaking to you, "Are you ready for war, <last>?"`
+			choice
+				`	"Not yet, I will come back when I am."`
+					goto later
+				`	"I was born ready."`
+			`	"And may you die ready, as well." Hi-Ye comes closer and takes you aside to speak to you, privately. "I have managed to buy myself a ship with the money from our last business opportunity. It is much stronger than the old one! And as a warlord, he has a strong fleet as well. You may want to consider this seriously. Do not forget you can call the fight off, to continue it later once you are ready, by giving us a small fee."`
+			choice
+				`	"You do not scare me."`
+					accept
+				`	"After thinking a bit... Maybe I should not take such risks, I will not participate in your war."`
+					decline
+			`	"That is good! Let us fight, on the same terms of honour as before. Take off when you will be ready, we will be waiting for you."`
+			label later
+			`	"Be sure to come back soon, war cannot wait forever."`
+				defer
+	
+	npc save disable
+		government "Hai (Unfettered Challenge)"
+		ship "Shield Beetle (Pulse)" "Intrepid II"
+		ship "Shield Beetle" "The Strategist"
+		conversation
+			action
+				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 100
+				event "shield beetles accessible"
+			`"You sure know how to fight, but sheer firepower may not be enough, you should train more before we are ready to start the war. Also we would require more jump drives that what we currently have, because our Alpha supplies have been disrupted by your actions."`
```

Comment:
```suggestion
			`"You sure know how to fight, but sheer firepower may not be enough, you should still train more before we are ready to start the war. Also we will require more jump drives than what we currently have, because our Alpha supplies have been disrupted by your actions."`
```